spec/models/teacher_spec.rb

```diff
@@ -8,4 +8,16 @@
     it { is_expected.to validate_presence_of(:email) }
     it { is_expected.to validate_presence_of(:password) }
   end
+
+  describe 'email validation fail' do
+    it 'for unique email' do
```

Comment:
Description of `it` should be written in the 3rd person. I also agree with [better specs here to keep description short](https://www.betterspecs.org/#short).

Keep in mind that suggestion below is very subjective - it's purely based on my habits :) 

```suggestion
    it 'validates uniqueness of the email' do
```

Reply:
Yes, I know that grammar thing about 'it'. I decided to take care of it later (because, as You know, I am no good in names), and have forgotten, as it often happens to me :(. Kick me in the ass some more - maybe that will help to improve my memory.
Check :)